app/controllers/bulk_downloads_controller.rb

```diff
@@ -2,8 +2,8 @@
 class BulkDownloadsController < ApplicationController
   include BulkDownloadTypesHelper
 
-  # GET /bulk_downloads/types
-  def types
-    render json: BulkDownloadTypesHelper::BULK_DOWNLOAD_TYPES
+  # GET /bulk_downloads/type_metadata
+  def type_metadata
```

Comment:
Seem to me that this is returning a list of download types, just like `projects`, `samples` etc. They all return metadata but we do not call them explicitly say its metadata.

Reply:
So do you think I should keep the original naming? I could do either. The reason I changed it was that I was running into a bit of confusion with "selectedDownloadType", which could either mean a string like "samples_taxon_summary" or the metadata object. I ran into a situation where I was using both in the same function and couldn't figure out how to name differently.

Reply:
I believe the most consistent would be to name as `selectedDownloadTypeName`. It is long but more accurate. And then you can have `downloadsTypes[selectedDownloadTypeName].<metadata_property>` or event `selectedDownloadType.<metadata_property>`

Reply:
Okay that makes sense. I will change to "type name" where there's a conflict..